src/components/DeckBuilderDryRunView/index.js

```diff
@@ -207,9 +207,33 @@ class DeckBuilderDryRunView extends React.Component {
           ? 'unit '
           : 'units '}
         on the board.
+        <br />
       </>
     )
   }
+
+  getFrozenEnemiesText = () => {
+    const { frozenEnemiesLevel } = this.props.specifics
+    const frozenStateDescriptionCount = {
+      0: 'no',
+      2: 'a few',
+      3: 'many',
+    }
+    const frozenStateDescription =
+      frozenEnemiesLevel === 4
+        ? 'The whole board is frozen.'
+        : frozenEnemiesLevel === 1
+        ? 'There is a frozen enemy on the board.'
+        : `There are ${frozenStateDescriptionCount[frozenEnemiesLevel]} frozen enemies on the board.`
+
+    return <>{frozenStateDescription}</>
+  }
+
+  containsFreeze = deckIds => {
+    const freezeCards = ['W1', 'W2', 'W4', 'W6', 'W8', 'W11']
```

Comment:
This goes by the assumption that Wisp Cloud, Zhevana and Icicle Burst are not included in a deck without freeze. :)

Reply:
The information is meant to be displayed whenever there are cards with an ability interacting with freeze. I think it can remain shown even when those 3 cards are in the deck

Reply:
If you don't think so, I'll remove them from the list

Reply:
Though it should still remain shown for Icicle

Reply:
I think what you did is good.